docs/topics/webhooks/index.mdx

```diff
@@ -0,0 +1,133 @@
+hide_title: true
+sidebar_position: 3
+sidebar_label: Webhooks
+sidebar_class_name: "sidebar-item-webhooks-overview"
+pagination_prev: null
+keywords:
+  - topics
+  - webhooks
+  - eda
+  - momento
+---
+
+## Use webhooks as subscribers to a topic
+
+Trigger endpoints in your application when a message is published to a topic. Every time an event is published, Momento will send a POST request to all configured webhooks and any other subscribers.
+
+## Why use webhooks?
+
+When using the Momento SDK to subscribe to a topic, a persistent connection is setup between your application code and the Momento service. This makes a standard subscription *stateful*, meaning the subscriber must always be on and available in order to publish or receive messages. Subscribers like browsers and containers work incredibly well in this environment, but it leaves stateless compute like Lambda functions or Google Cloud Run services out of luck.
+
+Webhooks allow you to subscribe any type of compute to receive events. Since a webhook does not maintain an active connection, you can put a serverless function behind an endpoint and be on your merry way!
+
+Using a webhook as a subscriber is perfect for asynchronous events like persisting messages to durable storage like a database, replicating data across regions, and even adding intermediary services like translation and moderation to chats.
+
+A topic can have both stateful subscriptions and webhooks. This allows you to connect anything you want, from anywhere, at any time!
+
+## The webhook event
+
+When an event is published to a topic, Momento will generate a request that includes metadata about the event in addition to the event payload. For enhanced security, this request includes additional headers.
+
+### Example event
+
+Below is a sample payload representing a message sent in a chat room. Note that this is a template and specific values will differ in your implementation.
+
+```json
+{
+  "cache": "chat",
+  "topic": "room-281",
+  "event_timestamp": 1704296470564,
+  "publish_timestamp": 1704296470565,
+  "topic_sequence_number": 1,
+  "token_id": "mo-the-squirrel",
+  "text": "Here I am!!"
+}
+```
+
+### Event structure
+
+The properties of the webhook event are listed below along with the function they provide.
+
+#### cache
+
+The name of the cache the topic belongs to. This is used for namespacing purposes in Momento Topics. All topics must be published to a valid cache in your account.
+
+#### topic
+
+The name of the topic that triggered the webhook. Remember, topic names are dynamic and are not resources in your Momento account. You can publish a message to any string, which adds an incredible amount of flexibility to messages, like using a variable as the topic name of your workflow.
+
+#### event_timestamp
+
+Timestamp in milliseconds since the Unix Epoch of when the event was published.
```

Comment:
To match the following, should remove "of": "...since the Unix Epoch when the event..."